azure-cli2017.pyproj

```diff
@@ -633,7 +636,23 @@
     <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\latest\test_storage_table_scenarios.py" />
     <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\latest\test_storage_url_helpers.py" />
     <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\latest\test_storage_validators.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\latest\__init__.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\storage_test_util.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_account_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_batch_operations.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_blob_copy_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_blob_live_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_blob_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_cors_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_file_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_queue_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_scenarios_non_ascii_chars.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_table_scenarios.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_url_helpers.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\test_storage_validators.py" />
+    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\tests\profile_2017_03_09\__init__.py" />
```

Comment:
Generally, we don't include the stack folders in the proj file so that people don't inadvertently edit the wrong files and instead just expose "latest".  When I need the stack folders, I use the show-all button.